backend/api/tests/views/test_views_datamanager.py

```diff
@@ -40,23 +40,23 @@ def setUp(self):
             os.makedirs(MEDIA_ROOT)
         self.url = reverse("api:data_manager-list")
         self.extra = {"HTTP_SUBSTRA_CHANNEL_NAME": "mychannel", "HTTP_ACCEPT": "application/json;version=0.0"}
+        self.maxDiff = None
```

Comment:
I think this should be removed, as it was used for debugging this specific function

Reply:
Ah yes you're right thanks !